### PR TITLE
Fixed build on window

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
> ### Motivate of the pull request
> - [ ] To address an existing issue. If so, please provide a link to the issue: <issue id>
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> 
> 
> 
> 
> 
>
> #### What does this pull request change?
> 
> 
> 
> 
> 

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

> ### Impact of the pull request

> - [ ] Require a change on Quality of Results (QoR)
> - [ ] Break back-compatibility. If so, please list who may be influenced.

### CODE REVIEW

File: src/Compiler/CMakeLists.txt [hunk 1]
```diff
@@ -100,7 +100,7 @@ endforeach()
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../bin)
 
 set(CMAKE_CXX_FLAGS_DEBUG
-"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Werror -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
+"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
```
Comment (alain-rs) [inline src/Compiler/CMakeLists.txt:103]: We want as many subsystems as possible to treat warnings as errors. So this is not a good change.

Comment (alain-rs) [reply src/Compiler/CMakeLists.txt:103]: Only when we did not write a subsystem and have little control on the code would we accept warnings.

File: src/Console/CMakeLists.txt [hunk 2]
```diff
@@ -98,7 +98,7 @@ install(
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/../../bin)
 
 set(CMAKE_CXX_FLAGS_DEBUG
-"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Werror -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
+"${CMAKE_CXX_FLAGS_DEBUG} ${TCMALLOC_COMPILE_OPTIONS} -Wall -O0 -g ${MSYS_COMPILE_OPTIONS} ${MY_CXX_WARNING_FLAGS} ${MEM_SANITIZER_FLAGS}")
```
[the remark this answers is not in the document]
Comment (KochynVolodymyr) [reply src/Console/CMakeLists.txt]: This flag affect gcc only. msvc failed on it. I moved it into the correct place.

File: src/Compiler/CMakeLists.txt (hunk 1 above)
Comment (KochynVolodymyr) [reply src/Compiler/CMakeLists.txt]: This flag affect gcc only. msvc failed on it. I moved it into the correct place.

**Pull request merged**